src/main/java/thePackmaster/cards/conjurerpack/Enervate.java

```diff
@@ -0,0 +1,41 @@
+package thePackmaster.cards.conjurerpack;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import thePackmaster.actions.conjurerpack.PlayRandomCardAction;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Enervate extends ConjurerCard
+{
+    public final static String ID = makeID(Enervate.class);
+    private static final int MAGIC = 2;
+
+
+    public Enervate() {
+        super(ID, 1, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.ENEMY);
+        baseMagicNumber = magicNumber = MAGIC;
+        exhaust = true;
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        if (m != null)
```

Comment:
Since this isn't in an action, there will be a few oddities -- for example, this will calculate strength for Gremlin Nob before  Nob's Enrage is applied. So if Nob is at 4 Strength with 3 Enrage:
* This sees Nob at 4 Strength and locks in -4 strength
* Nob gains 3 Strength, going up to 7
* Nob loses 4 Strength, going down to 3

It would be more consistent if this worked like so:
* This card is played
* Nob gains 3 Strength, going up to 7
* This applies -5 Strength to Nob, going down to 2

You can achieve this by changing this code to the following:
```
        this.addToBot(new AbstractGameAction() {
            @Override
            public void update() {
                if (m != null)
                {
                    int decAmount = magicNumber;
                    AbstractPower strPower = m.getPower(StrengthPower.POWER_ID);
                    if (strPower != null && strPower.amount > 0)
                    {
                        decAmount += strPower.amount / 2;
                    }
                    this.addToTop(new ApplyPowerAction(m, p, new StrengthPower(m, -decAmount), -decAmount));
                }
                this.isDone = true;
            }
        });
```